### PR TITLE
Added Qtum.

### PR DESCRIPTION
**Pull request by dooglio**
Added what I believe to be the basis of Qtum.

### CODE REVIEW

File: lib/coins/qtum.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  // base58Prefixes
+  versions: {
+    bip32: {
+      private: 0x0488ade4,
+      public: 0x0488b21e
+    },
+    bip44: 88,
+    private: 0x80,
+    public: 0x26,
+    scripthash: 0x17
```
Comment (fanatid) [inline lib/coins/qtum.js]: https://github.com/qtumproject/qtum/blob/master/src/chainparams.cpp#L138-L139
public & scripthash params not looks same

**Comment by RyanZim** [comment]
Still unsure on the bip44 value; waiting for a response from the QTUM team.

**Comment by fanatid** [comment]
We received response from QTUM team, they chose 88 as constant for BIP44 and recommended to us use it.
Unfortunately this is already taken from other coin, ref: https://github.com/satoshilabs/slips/pull/120

**Comment by fanatid** [comment]
Released as 3.1.0

**Pull request merged**